bundles/com.espressif.idf.core/src/com/espressif/idf/core/SystemWrapper.java

```diff
@@ -0,0 +1,11 @@
+/*******************************************************************************
+ * Copyright 2023 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+package com.espressif.idf.core;
+
+public interface SystemWrapper
```

Comment:
Same for this one it would be better to have the interface names starting with **I** 
`ISystemWrapper`

Reply:
"I" prefix is not a standard naming convention in Java for interfaces. Java standard interfaces as a reference:
Map, List, Collection, etc.